plugins/reminders.py

```diff
@@ -65,8 +65,9 @@ async def send_reminder(reminder: Reminder) -> None:
             "Reminder {} for user {} silently removed (guild no longer exists)".format(reminder.id, reminder.user_id)
         )
         return
-    channel = await guild.fetch_channel(reminder.channel_id)
-    if not isinstance(channel, (TextChannel, Thread)):
+    try:
+        assert isinstance(channel := await guild.fetch_channel(reminder.channel_id), (TextChannel, Thread))
+    except (discord.NotFound, AssertionError):
```

Comment:
Assertions are supposed to be bugs in the program. Please just keep the `if` and insert a `try` around `fetch_channel` with a separate log message